src/components/ebay-textbox/index.js

```diff
@@ -55,9 +55,19 @@ function getTemplateData(state) {
     return state;
 }
 
-function init() {
-    if (this.state.floatingLabel) {
-        this.floatingLabel = new FloatingLabel(this.el);
+function onRender() {
+    if (this.state.floatingLabel && !this.floatingLabel && document.readyState === 'complete') {
+        this.initFloatingLabel(this);
+    } else if (this.state.floatingLabel) {
+        window.addEventListener('load', this.initFloatingLabel(this));
+    }
+}
+
+function initFloatingLabel(_this) {
```

Comment:
Since we are using `this.initFloatingLabel` we shouldn't need to pass the `_this` parameter (just use this as is).

Reply:
I can't reproduce the testing issue which caused me to do this. Reverting.